packages/tasks/src/tasks/visual-document-retrieval/data.ts

```diff
@@ -0,0 +1,71 @@
+import type { TaskDataCustom } from "../index.js";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "A large dataset used to train visual document retrieval models.",
+			id: "vidore/colpali_train_set",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "input.png",
+				type: "img",
+			},
+			{
+				label: "Question",
+				content: "Is the model in this paper the fastest for inference?",
```

Comment:
Judging from the outputs, should the question be something about where in the document can the answer be found?

Reply:
Yes! But if I I'm understanding correctly the outputs in this example are the places where the answer can be found, not the answer itself, that's why I was asking :)

Reply:
yes, that's what you pass to VLM to get the final answer! (in multimodal RAG, after you get the retrieved page, you pass the page along with question to VLM)

it gives you the page only, at best you can visualize which image patches are most activated but other than that, you only get the page number. 
<img width="387" alt="Screenshot 2025-04-24 at 15 07 45" src="https://github.com/user-attachments/assets/8566beb4-43dc-42b2-9cc2-0cbb0fae710c" />

did I understand your concern properly?

Reply:
Sounds good!